conda/cli/main_config.py

```diff
@@ -365,6 +365,29 @@ def execute_config(args, parser):
                                 (item, key))
         rc_config[key] = [i for i in rc_config[key] if i != item]
 
+    # Clear
+    for key, in args.clear:
+        key, subkey = key.split('.', 1) if '.' in key else (key, None)
+        if key == 'channels' and key not in rc_config:
+            rc_config[key] = ['defaults']
```

Comment:
This hardcoded value assumes the user started with `Miniconda`, which hardcodes `defaults`. However, other distributions like Miniforge or Mambaforge hardcode `conda-forge`. There might be other distros out there with different configurations too.

Reply:
ah I see- would it make sense to just remove this default logic, since this `clear` option would be emptying arrays versus returning to any default value?

Reply:
Let's go ahead and drop this special conditional